.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -669,6 +669,32 @@ popd > /dev/null
 chown -R ${DAEMON_USER}:${COURSE_BUILDERS_GROUP} ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools
 chmod -R 555 ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools
 
+################################################################################################################
+################################################################################################################
+# BUILD AND INSTALL ANALYSIS TOOLS TS
+
+echo -e "Build and install analysis tools ts"
+
+mkdir -p "${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS"
+
+ANALYSIS_TOOLS_TS_REPO="${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT/AnalysisToolsTS/"
+
+# Copy cloned files to AnalysisToolsTS directory
+rsync -rtz "${ANALYSIS_TOOLS_TS_REPO}" "${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS"
+
+pushd "${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS"
+
+# # change permissions
+chown -R "${DAEMON_USER}":"${COURSE_BUILDERS_GROUP}" "${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS"
+chmod -R 755 "${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS"
+
+# # install npm packages
+su - ${DAEMON_USER} -c "cd ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS && npm install"
+
+# # build project
+su - ${DAEMON_USER} -c "cd ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS && npm run build"
```

Comment:
These variables should ideally be quoted.  Additionally, is there a reason why the `cd` is in these statements?  I think it would be better to have separate `pushd` and `popd` statements above and below each of the `su` statements, unless there's a reason for the current behavior.

Reply:
Using `su -` changes the currently residing directory that to the switched users home. Thus the `cd` was needed there. . Changed the command to use `su` so the directories will not get changed.

Is it okay to use `su` instead of `su -` (from security point).  Because following article says.
https://www.geeksforgeeks.org/difference-between-su-and-su-command-in-linux/

"_su command does not create a new User Environment (in the simple term they pretends to be the target user) but su - creates a totally new User Environment (in the simple term they are actually the target user) that we can check with environment variables like:_"

Reply:
Also based on the meeting yesterday removed the addition of `submitty_count_ts` command in execute.cpp. Will add this in a seperate PR.